.github/dependabot.yml

```diff
@@ -9,4 +9,4 @@ updates:
   - package-ecosystem: "npm"
```

Comment:
👆🏻 Noticed this is only for the Next.js platform

Reply:
> should this file go into template-application-nextjs instead of being in template-infra?

Good question. I am tempted to do something similar as what we did with the CodeQL workflow, which is to not include it in the template. GitHub provides CodeQL and Dependabot workflows when you enable those features on a repo's settings, and that may be a better way to set these up since it will be catered to the repo (i.e if it is a monorepo with JS and Python, the workflow GitHub generates will support both).

Reply:
that sounds reasonable to me. so to confirm my understanding, the process would be:

1. create a repo
2. copy all the templates you need (flask, nextjs)
3. enable the codeql or dependabot setting in the repo, at that time github will generate based on what's in the repo

if that's right should we instead just update the instructions in this repo rather than including the configured dependabot file?

Reply:
Yep, that sounds right to me. I can update this branch to reflect this new direction.